### DISCUSSION TITLE
5.2.2.0

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#343] [Announcements]
- fix app not starting with generic drivers (and maybe others)
- dont start minimised at first install
- fix border resistance coherence
- fix process priority not working
- fix memory corruption crash

## Donation
If this project help you, you can give me a cup of coffee :) 
[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=YLGYPSHWTQ5UW&lc=FR&item_name=Mgth&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

**Full Changelog**: https://github.com/mgth/LittleBigMouse/compare/v5.2...v5.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mgth/LittleBigMouse/releases/tag/v5.2.2'>5.2.2.0</a>.</em>

**Comment by SnoWFLakE0s** [comment]
Downgraded to this version from latest; unfortunately it seems more reliable.

**Comment by ErichJu88** [comment]
Gemini hat mir empfohlen nicht die Version 5.3 zu installieren aber die v5.2.2 oder v5.2.2.0 existiert kein download und keine exe!
Danke bitte eine antwort an ahc-hotel@t-online.de Danke
